qutebrowser/browser/downloads.py

```diff
@@ -742,15 +766,17 @@ def _after_create_parent_question(self,
                    "re-download?".format(html.escape(self._filename)))
             self._ask_confirm_question(
                 "Cancel other download?", txt,
-                custom_yes_action=self._cancel_conflicting_download)
+                custom_yes_action=self._cancel_conflicting_download,
+                custom_no_action=self._find_next_filename)
         elif force_overwrite:
             self._after_set_filename()
         elif os.path.isfile(self._filename):
             # The file already exists, so ask the user if it should be
             # overwritten.
-            txt = "<b>{}</b> already exists. Overwrite?".format(
+            txt = "<b>{}</b> already exists. Overwrite? (\"No\" renames)".format(
```

Comment:
Could you add this bit to the "conflicting download" prompt which is also seeing a change in behavior too please? Also think about if it needs a full stop on the end since it's amongst a bunch of full sentences.